src/autocomplete/EmojiProvider.js

```diff
@@ -69,12 +66,16 @@ export default class EmojiProvider extends AutocompleteProvider {
     constructor() {
         super(EMOJI_REGEX);
         this.matcher = new QueryMatcher(EMOJI_SHORTNAMES, {
-            keys: ['aliases_ascii', 'shortname', 'aliases'],
+            keys: ['emoji.emoticon', 'shortname'],
+            funcs: [
+            //     (o) => `:${o.emoji.shortcodes[0]}:`, // shortname
```

Comment:
is this not needed?

Reply:
either this or https://github.com/matrix-org/matrix-react-sdk/pull/3745/files#diff-81bd9c47d9cc4384c17cd19db915083fR50 is needed but given its accessed in a couple more places later down I thought I'd just do the concat once. But yes this commented line should be gone